autopush_rs/src/client.rs

```diff
@@ -990,22 +994,26 @@ where
     ) -> Poll<AfterAwaitUnregister<T>, Error> {
         debug!("State: AwaitUnRegister");
         let msg = match try_ready!(await_unregister.response.poll()) {
-            call::UnRegisterResponse::Success { success } => {
+            true => {
```

Comment:
since this is a bool now, I think you'd save a few lines of code making this an if/else instead of a match. especially since both blocks return ServerMessage::Unregister